### DISCUSSION TITLE
Local fonts

### DISCUSSION THREAD

**Discussion by ahoiroman** [lucky-media/cloud#4] [General]
Hello,

as you can see [here](https://github.com/lucky-media/cloud/blob/main/resources/views/layout.antlers.html#L12-L14) the fonts are being loaded from a google-domain. Unfortunately this is a bit dangerous when it comes to GDPR. It would be great if it would be possible to serve the fonts from local storage (see https://google-webfonts-helper.herokuapp.com/fonts).

**Comment by lokmanm** [comment]
Hey @ahoiroman , we use [fontsource](https://github.com/fontsource/fontsource) for our projects as its a bit easier to include them.
Font Source does the same job as the `google-webfonts-helper` but its a bit more convenient.

We will try to fix this along with the dependency updates the following days.

Again, thanks for the suggestions.

**Reply by lokmanm** [reply]
This is now fixed with the latest release of [v1.1](https://github.com/lucky-media/cloud/releases/tag/v1.1)

**Comment by ahoiroman** [comment]
Great. So now I can use that starter-kit 😊 I‘m going to buy it right now 😊

**Reply by lokmanm** [reply]
Thank you !
